apps/profiles/pipeline.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib.auth import get_user_model
 from django.core.exceptions import ObjectDoesNotExist
+from .models import GithubUserInfo
```

Comment:
Change to fullpath import:
`from apps.profiles.models import GithubUserInfo`

Reply:
Done.